carbon-design-system/carbon-addons-iot-react
# fix(dashboard): fix small card sizes

Closes #2370 

**Summary**

- Adjusted small card sizes:
- `SMALL` is now 2 wide on a `md` breakpoint
- `SMALLWIDE` is now only 1 tall on a `sm` breakpoint 
- `SMALLWIDE` is now 4 wide on a `md` breakpoint
- `SMALLFULL` is not only 1 tall on a `sm` breakpoint

**Acceptance Test (how to verify the PR)**

- Open the dashboard editor with breakpoint switcher story 
- set it to the medium breakpoint
- add a SMALL card and it should be 1/4 width and 1 tall
- switch to the small breakpoint
- it should still be 1/4 width and 1 tall.
- Change it to a SMALLWIDE card.
- it should be 1 tall and half the width
- switch it to SMALLFULL and it should still be 1 tall, but take the full width

**Regression Test (how to make sure this PR doesn't break old functionality)**

- no other changes



**Things to look for during review**
